apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -112,9 +112,17 @@ export class FilesStorageController {
 
 		const streamableFile = this.streamFileToClient(req, fileResponse, response, bytesRange);
 
+		this.setAttachmentDispositionHeaderForNonPDF(fileResponse, response);
+
 		return streamableFile;
 	}
 
+	private setAttachmentDispositionHeaderForNonPDF(fileResponse: GetFileResponse, response: Response): void {
+		if (fileResponse.contentType !== 'application/pdf') {
```

Comment:
Should we maybe add a comment here explaining why we do it? Like: "We set this header for any non-PDF files to avoid XSS attacks with malicious files (e.g. HTML files). We whitelist PDF files here because it is the only file type that should be opened directly in the browser."

Reply:
Okay, that's a valid point. Also not sure then...